cmd/cli.go

```diff
@@ -77,6 +84,65 @@ var dumpCmd = &cobra.Command{
 	},
 }
 
+var containerdDumpCmd = &cobra.Command{
+	Use:   "containerd",
+	Short: "Manually checkpoint a running container to a directory",
+	Args:  cobra.ArbitraryArgs,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		cli, err := NewCLI()
+		if err != nil {
+			return err
+		}
+
+		a := api.ContainerDumpArgs{
+			Ref:         ref,
+			ContainerId: containerId,
+		}
+
+		var resp api.ContainerDumpResp
+		err = cli.conn.Call("CedanaDaemon.ContainerDump", a, &resp)
+		if err != nil {
+			return err
+		}
+		cli.logger.Info().Msgf("container %s dumped successfully to %s", containerId, dir)
```

Comment:
Is it actually checkpointing it to a directory? where are the image files stored?

Reply:
It's checkpointing to /tmp/ctd-{uuid}. That's if you don't specify where to unpack. I will add an optional flag for checkpoint directory in the future

Reply:
`	if image == "" {
		checkpointImageExists = true
		image, err = os.MkdirTemp(os.Getenv("XDG_RUNTIME_DIR"), "ctrd-checkpoint")
		if err != nil {
			return &apiTasks.CheckpointTaskResponse{}, err
		}
		criuOpts.ImagesDirectory = image
		fmt.Printf("Checkpointing to %s\n", image)
		defer os.RemoveAll(image)
	}`